include/swift/AST/ASTScope.h

```diff
@@ -30,6 +30,9 @@
 
 #include "swift/AST/ASTNode.h"
 #include "swift/AST/NameLookup.h" // for DeclVisibilityKind
+// extractNearestSourceLoc, simple_display declarations must come before this
+// Alternatively, declare these in SimpleDisplay.h or elsewhere
+//#include "swift/AST/SimpleRequest.h"
```

Comment:
You should be able to include `SimpleRequest.h` here with a change a note below, then you can remove the comment.